lightspeed-catalog-4.15/index.yaml

```diff
@@ -63,3 +114,12 @@ name: preview
 entries:
   - name: lightspeed-operator.v0.1.4
     skipRange: ">=0.1.0 <0.1.4"
+---
+schema: olm.channel
+package: lightspeed-operator
+name: alpha
+entries:
+  - name: lightspeed-operator.v0.1.3
+    skipRange: ">=0.1.0 <0.1.3"
+  - name: lightspeed-operator.v0.1.4
+    replaces: lightspeed-operator.v0.1.3
```

Comment:
What are we gaining in this definition  than just specifying `skipRange: ">=0.1.0 <0.1.4" @raptorsun

Reply:
if we pass only on snapshot, it will list a single item and skip all older versions